services/provider/server/storageclaim.go

```diff
@@ -66,8 +67,8 @@ func (s *storageClassClaimManager) Create(ctx context.Context, consumer *ocsv1al
 			Namespace:       s.namespace,
 			OwnerReferences: []metav1.OwnerReference{ownerRef},
 			Labels: map[string]string{
-				consumerUUIDLabel:          consumerUUID,
-				storageClassClaimNameLabel: storageClassClaimName,
+				controllers.ConsumerUUIDLabel: consumerUUID,
```

Comment:
Are you sure that It will work, I think we did this earlier and we were having cyclic dependencies? Try to build a container using `make ocs-operator`